crest/Assets/Crest/Crest/Scripts/LodData/RegisterLodDataInput.cs

```diff
@@ -53,12 +53,15 @@ protected virtual void Start()
             }
         }
 
-        public void Draw(CommandBuffer buf, float weight, int isTransition)
+        public void Draw(CommandBuffer buf, float weight, int isTransition, int lodIdx)
         {
             if (_renderer && weight > 0f)
             {
                 _materials[isTransition].SetFloat(sp_Weight, weight);
 
+                _materials[0].SetInt(LodDataMgr.sp_LD_SliceIndex, lodIdx);
```

Comment:
would only this suffice:
`_materials[isTransition].SetInt(LodDataMgr.sp_LD_SliceIndex, lodIdx);`

Reply:
Thanks